XenditExample/XenditExample/Controllers/CreateAuthentification.swift

```diff
@@ -28,15 +32,21 @@ class CreateAuthentification: UIViewController {
                 
         let tokenID = tokenIDTextField.text
         let cardCVN = cardCVNTextField.text
+        let cardData = XenditCardHolderInformation.init(
+            cardHolderFirstName: cardHolderFirstNameTextField.text,
+            cardHolderLastName: cardHolderLastNameTextField.text,
+            cardHolderEmail: cardHolderEmailTextField.text,
+            cardHolderPhoneNumber: cardHolderPhoneNumberTextField.text
+        )
         
-        if tokenID == nil, amountTextField.text == nil, cardCVN == nil {
+        if tokenID == nil, amountTextField.text == nil, cardCVN == nil, cardData == nil {
```

Comment:
I thought this part was updated? Because this only checks if variable is nil or not, rather than if there is content or not